metashare/api/gh.py

```diff
@@ -137,6 +139,7 @@ def extract_zip_file(zip_file, owner, repo_name):
     for path in itertools.chain(glob("zipball_root/*"), glob("zipball_root/.*")):
         shutil.move(path, ".")
     shutil.rmtree("zipball_root")
+    shutil.rmtree(ZIP_FILE_NAME)
```

Comment:
Maybe should be `os.remove`?

```
HTTP POST /api/scratch-orgs/o1e03y8/commit/ 202 [0.05, 127.0.0.1:55980]
Traceback (most recent call last):
  File "/app/metashare/api/jobs.py", line 168, in commit_changes_from_org
    commit_message=commit_message,
  File "/app/metashare/api/sf_org_changes.py", line 58, in commit_changes_to_github
    with local_github_checkout(user, repo_url) as project_path:
  File "/usr/local/lib/python3.7/contextlib.py", line 112, in __enter__
    return next(self.gen)
  File "/app/metashare/api/gh.py", line 161, in local_github_checkout
    extract_zip_file(zip_file, owner, repo_name)
  File "/app/metashare/api/gh.py", line 142, in extract_zip_file
    shutil.rmtree(ZIP_FILE_NAME)
  File "/usr/local/lib/python3.7/shutil.py", line 491, in rmtree
    _rmtree_safe_fd(fd, path, onerror)
  File "/usr/local/lib/python3.7/shutil.py", line 410, in _rmtree_safe_fd
    onerror(os.scandir, path, sys.exc_info())
  File "/usr/local/lib/python3.7/shutil.py", line 406, in _rmtree_safe_fd
    with os.scandir(topfd) as scandir_it:
NotADirectoryError: [Errno 20] Not a directory: 'archive.zip'
```

Reply:
Should be; I thought that call was flexible. Will fix.

Reply:
After making that change locally, now I get:

```
Traceback (most recent call last):
  File "/app/metashare/api/jobs.py", line 168, in commit_changes_from_org
    commit_message=commit_message,
  File "/app/metashare/api/sf_org_changes.py", line 63, in commit_changes_to_github
    user, scratch_org, os.path.join(project_path, "src"), desired_changes
  File "/app/metashare/api/sf_org_changes.py", line 43, in run_retrieve_task
    "commit": branch,
  File "/usr/local/lib/python3.7/site-packages/cumulusci/core/runtime.py", line 32, in __init__
    self._load_project_config(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/cumulusci/core/runtime.py", line 82, in _load_project_config
    self.global_config, *args, **kwargs
  File "/usr/local/lib/python3.7/site-packages/cumulusci/core/config/BaseProjectConfig.py", line 48, in __init__
    super(BaseProjectConfig, self).__init__(config=config)
  File "/usr/local/lib/python3.7/site-packages/cumulusci/core/config/BaseConfig.py", line 15, in __init__
    self._load_config()
  File "/usr/local/lib/python3.7/site-packages/cumulusci/core/config/BaseProjectConfig.py", line 75, in _load_config
    self.config_filename, repo_root
cumulusci.core.exceptions.ProjectConfigNotFound: The file cumulusci.yml was not found in the repo root: /tmp/tmp2zbduj9i/src. Are you in a CumulusCI Project directory?
```